Instructions/GitHubProjects.md

```diff
@@ -0,0 +1,127 @@
+
+As the week progresses, tasks get moved from the "To Do" column, to the "In Progress" column, and then finally to the "Done" column. This helps teams take stock of what was accomplished and identify what is left to do. Don't worry if not everything gets done! The board is just supposed to be a guideline, and tasks often take either longer or shorter than we expect. Regular communication with your team can help ensure that you are getting the right amount of work done for part of the project.
+
+> [!NOTE]
+> Since the main focus of this course is on Object-Oriented programming and not Software Development, you don't need to worry too much about how structured your team's process is for managing your project board. It can be as informal as adding new items and then letting your other team members know or setting aside the first few minutes of group meetings to go over the board. The key part of having this set up is that you are using it as a way to help track and document the work on your project. This will help you, your team, and the TAs track the overall work and progress that your team is making on your project.
+
+
+### Adding group members to your GitHub Project
+
+This section describes how to manage access to your GitHub Project and add your team members.
+
+> [!IMPORTANT]
+> By default, only the person who creates the GitHub Project has access to add, move, or delete items from the project board. For your group project, you will want to make sure that you add everyone in your group to the project (including the TA responsible for overseeing your group). Although the TA will not make any edits to your team’s board directly, adding them to the project will allow them to see the tasks that your team includes and track your team’s progress.
+
+**To add a new member to your project, follow these steps:**
+
+1. Click on the Project settings from the menu on the upper right-hand side of the Project window (the three dots icon).
+
+<br>
+
+<img src="images/project_settings.png" alt="GitHub Logo" width="300">
+
+<br>
+
+2. Search for your team member by thier GitHub username and invite them to collaborate
+
+<br>
+
+<img src="images/project_invite.png" alt="GitHub Logo" width="800">
+
+<br>
+
+> [!NOTE]  
+> You can manage a user’s role (permissions) to the Project by clicking on the roles dropdown. The TA for your group should be given Read access to view the project 
```

Comment:
TAs may already have this as you are admins to everything.. It may be worth testing with Group 4, since they are ahead of the game.

Reply:
Looks like there is some visibility into projects (but will need to ultimately confirm once student has enabled project board)